neo/Prompt/Commands/Config.py

```diff
@@ -0,0 +1,205 @@
+                settings.set_emit_notify_events_on_sc_execution_error(True)
+                print("Smart contract emit Notify events on execution failure is now enabled")
+                return c1
+            elif c1 == 'off' or c1 == '0':
+                settings.set_emit_notify_events_on_sc_execution_error(False)
+                print("Smart contract emit Notify events on execution failure is now disabled")
+                return c1
+            else:
+                print("Cannot configure log. Please specify on|off")
+                return
+        else:
+            print("Cannot configure log. Please specify on|off")
+            return
+
+    def command_desc(self):
+        p1 = ParameterDesc('attribute', 'either "on"|"off" or 1|0')
+        return CommandDesc('sc-debug-notify', 'toggle Notify events on execution failure on or off', [p1])
+
+
+class CommandConfigVMLog(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        c1 = get_arg(arguments)
+        if c1 is not None:
+            c1 = c1.lower()
+            if c1 == 'on' or c1 == '1':
+                settings.set_log_vm_instruction(True)
+                print("VM instruction execution logging is now enabled")
+                return c1
+            elif c1 == 'off' or c1 == '0':
+                settings.set_log_vm_instruction(False)
+                print("VM instruction execution logging is now disabled")
+                return c1
+            else:
+                print("Cannot configure VM instruction logging. Please specify on|off")
+                return
+        else:
+            print("Cannot configure VM instruction logging. Please specify on|off")
+            return
+
+    def command_desc(self):
+        p1 = ParameterDesc('attribute', 'either "on"|"off" or 1|0')
+        return CommandDesc('vm-log', 'toggle VM instruction execution logging on or off', [p1])
+
+
+class CommandConfigNodeRequests(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        if len(arguments) in [1, 2]:
+            if len(arguments) == 2:
+                try:
+                    return NodeLeader.Instance().setBlockReqSizeAndMax(int(arguments[0]), int(arguments[1]))
+                except ValueError:
+                    print("Please specify a block request part and max size for each node, like 30 and 1000")
+                    return False
+            elif len(arguments) == 1:
+                return NodeLeader.Instance().setBlockReqSizeByName(arguments[0])
+        else:
+            print("Invalid number of arguments")
+            return False
+
+    def command_desc(self):
+        p1 = ParameterDesc('attribute', 'specify "slow"/"normal"/"fast" preset block request sizes, or specific block request size')
+        p2 = ParameterDesc('queue-size', 'specific max queue size. Required if custom block request size is specified', optional=True)
```

Comment:
2 things; 
1. the description isn't clear on what queue specifically, so a user will have a hard time understanding what changing this setting will have for an effect. I personally had to read the code to understand the purpose because I've not used this command before.
2. it's odd that the item is flagged as optional but the description says mandatory. 

I think we should 
1. change the description to something like `The maximum number of outstanding block requests`
2. we should change https://github.com/CityOfZion/neo-python/blob/e3780126f9cf8175f9f3c7ad27f36f2baae20508/neo/Network/NodeLeader.py#L137-L143
to have default values equal to the normal settings (`100`/`10000`) and a maximum of 500 for `breqpart` (as pointed out in the NodeLeader comment above). 

In the most ideal situation we would probably have a custom help printer for this command indicating that only one of 2 params is mandatory. We could create an enhancement issue for this for a later point in time. For now I'd say just make them both mandatory. We could help to user by printing the current settings.

Reply:
@ixje When would you print the current settings? Or would you add this as another option e.g. `config node-requests current`?

Reply:
Easiest is probably to add the following in `CommandConfigNodeRequests`
```
    def handle_help(self, arguments):
        super().handle_help(arguments)
        print(f"\nCurrent settings {self.command_desc().params[0].name}: {NodeLeader.Instance().BREQPART}  {self.command_desc().params[1].name}: {NodeLeader.Instance().BREQMAX}")
```

Reply:
That is awesome 😄